tests/meltano/core/logging/test_output_logger.py

```diff
@@ -37,6 +37,10 @@ def fixture_configure_structlog(self, log_output):
             logger_factory=structlog.stdlib.LoggerFactory(),
             wrapper_class=structlog.stdlib.BoundLogger,
         )
+        try:
+            yield
+        finally:
+            structlog.reset_defaults()
```

Comment:
Ideally we would reset to whatever was previously configured, rather than the default.